guides/overview-apps.md

```diff
@@ -1,3 +1,28 @@
-# Apps Made by Wix
+# About Apps Made by Wix
 
-Wix offers a range of powerful business solutions that extend the functionalities of its existing products or add new capabilities. Apps made by Wix are web apps created by Wix that site owners can use to enhance their sites. A site owner can install one or more apps made by Wix to manage specific data on their site. Each app is identified by a unique `appId`. See a complete list [here](https://dev.wix.com/docs/develop-websites/articles/wix-apps/about-apps-made-by-wix).
\ No newline at end of file
+Wix offers a range of powerful business solutions that extend the functionalities of its existing products or add new capabilities. Apps made by Wix are web apps created by Wix that site owners can use to enhance their sites. A site owner can install one or more apps made by Wix to manage specific data on their site. 
+
+You may want to access site data that is managed by an app made by Wix. When making function calls to access Wix site data, some function calls require a reference to the specific app made by Wix. For example, to add a comment with the [`createComment()`](https://www.wix.com/velo/reference/wix-comments-v2/comments/createcomment) function, pass an `appId` to specify where to create the comment, such as in Wix Blogs. 
+
+Other function calls may return a reference to the app made by Wix. For example, calling the [`getCoupon()`](https://www.wix.com/velo/reference/wix-marketing-v2/coupons/getcoupon) function returns a coupon object with an `appId` referring to the ID of the Wix app that created the coupon, such as Wix Bookings or Wix Events. 
```

Comment:
Starting at line 5 with the second instance of it and on, you can probably start referring to the app made by Wix as a Wix app. I think it will flow better.

Reply:
I would also write " appId **that refers to** the ID of"